### PR TITLE
Merge rhel9 branch into master

### PR DESCRIPTION
**Pull request by matejak**
The RHEL9 got some fixes that we want in master as well.

In another PR, we will take goodies that got into the master, trying to propagate them to RHEL9.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,15 +1,25 @@
 NAME = oscap-anaconda-addon
 
-VERSION = 3.0.0
+VERSION = 0.35.0
```
Comment (matejak) [inline Makefile:3]: I propose to use versions targetting Fedora in the format `0.<fedora version>.<patch version component>`

File: org_fedora_oscap/common.py [hunk 2]
```diff
@@ -45,8 +45,10 @@
 from pyanaconda.modules.common.constants.services import NETWORK, PAYLOADS
 from pyanaconda.modules.common.structures.payload import PackagesSelectionData
```
Comment (matejak) [inline org_fedora_oscap/common.py:46]: This needs investigation, it seems to be incorrect.

**Comment by matejak** [comment]
So this PR works to a degree that the spoke comes up, but I get an error `DBus object has not attribute 'Packages'.` on F35. @poncovka is this expected and should we aim to F36, or is there more work to do than just pulling #172 to this PR?

**Comment by poncovka** [comment]
> So this PR works to a degree that the spoke comes up, but I get an error `DBus object has not attribute 'Packages'.` on F35. @poncovka is this expected and should we aim to F36, or is there more work to do than just pulling #172 to this PR?

I think it needs https://github.com/OpenSCAP/oscap-anaconda-addon/pull/179 for both, F35 and F36. I have tried to run the unit tests, but I don't have the patience to cherry-pick commits from pull requests with fixes anymore, so it is not tested.

**Comment by matejak** [comment]
> I think it needs #179 for both, F35 and F36. I have tried to run the unit tests, but I don't have the patience to cherry-pick commits from pull requests with fixes anymore, so it is not tested.

Thanks, I have integrated that PR into this merge PR, and it works - the F35 installation with hardening finishes successfully.
Therefore, the PR can be reviewed by our team and hopefully merged.

**Comment by evgenyz** [comment]
Okay, you fixed tests? and it is great. But I honestly can't imagine how I could possibly review this leviathan.

**Comment by evgenyz** [comment]
> Thanks, I have integrated that PR into this merge PR, and it works - the F35 installation with hardening finishes successfully. Therefore, the PR can be reviewed by our team and hopefully merged.

I mean, formally I see that it works and passes test. I guess, since we are not shipping master anywhere we can consider it as good enough to merge and continue working. Would that be a good judgement?

**Comment by jan-cerny** [comment]
There are some commits after the "merge" commits. How is that possible?

**Comment by matejak** [comment]
> There are some commits after the "merge" commits. How is that possible?

They were required to make the addon work with the current Fedora. It is IMO better to work with PRs that result in working software that can be tested functionally. A pure merge commit would probably work on Fedora 34, but it wouldn't pass the Github gating, and technically the master branch is supposed to work with rawhide.

**Comment by jan-cerny** [comment]
great!

**Review by jan-cerny — APPROVED** [review]

**Pull request merged**